server/handlers/destinations.go

```diff
@@ -172,13 +249,73 @@ func testDestinationConnection(config *storages.DestinationConfig) error {
 		fbAdapter := adapters.NewTestFacebookConversion(config.Facebook)
 
 		return fbAdapter.TestAccess()
+
 	case storages.WebHookType:
 		if err := config.WebHook.Validate(); err != nil {
 			return err
 		}
 
 		return nil
+
 	default:
-		return errors.New("unsupported destination type " + config.Type)
+		return fmt.Errorf("Unsupported destination type: '%v'", config.Type)
 	}
 }
+
+func TestProcessing(config *storages.DestinationConfig) error {
```

Comment:
It would be better to give a name like `TestBatch`

Reply:
Changed to `TestBatchProcessing`